ConnectMessage/index.js

```diff
@@ -1,175 +1,165 @@
-/**

```

Comment:
I think we should probably keep this header.

Reply:
I was trying to highlight all of this. Other than that, LGTM. 

* @file index.js -- Azure Function Node.js function.
 * This function receives a DocuSign Connect notification message
 * and enqueues it to an Azure Service Bus queue.